include/crow/http_response.h

```diff
@@ -29,6 +29,8 @@ namespace crow
         std::string body; ///< The actual payload containing the response data.
         ci_map headers; ///< HTTP headers.
         bool compressed = true; ///< If compression is enabled and this is false, the individual response will not be compressed.
+        bool head = false; ///< Whether this is a response to a HEAD request.
```

Comment:
Maybe better names for these variables (`head` and ` no_length`)? I had to check what are these here, in comment - in code they were pretty vague. Maybe something like `is_head_response` or `is_response_to_head_request`? And `add_automatically_content_length` or something similar?